erpnext/accounts/doctype/accounts_settings/accounts_settings.json

```diff
@@ -368,14 +370,26 @@
    "fieldname": "book_tax_discount_loss",
    "fieldtype": "Check",
    "label": "Book Tax Loss on Early Payment Discount"
+  },
+  {
+   "fieldname": "journals_section",
+   "fieldtype": "Section Break",
+   "label": "Journals"
+  },
+  {
+   "default": "1",
```

Comment:
Defaults won't be set on existing site btw.

Reply:
@ruthra-kumar You can add a patch to update the value for this

Reply:
Patch added